purchases-js-hybrid-mappings/package.json

```diff
@@ -30,7 +30,7 @@
     "allchecks": "npm run format && npm run lint:fix && npm run api-test && npm run test"
   },
   "dependencies": {
-    "@revenuecat/purchases-js": "1.13.2"
+    "@revenuecat/purchases-js": "1.14.0"
```

Comment:
Hmm we could trigger an update of the natives here instead of just updating purchases-is… you can do that from CircleCI dependency-update action